src/components/chats/ContactInfo/index.vue

```diff
@@ -1,227 +1,225 @@
+            </div>
+            <div
+              style="display: flex; margin-left: -8px; align-items: center"
+              v-if="!isLinkedToOtherAgent && !isViewMode && !isHistory"
+            >
+              <unnnicSwitch
+                :value="isLinkedUser"
+                @input="addContactToAgent"
+                size="small"
+                :textRight="
+                  isLinkedUser
+                    ? $t('contact_info.switch_disassociate_contact')
+                    : $t('contact_info.switch_associate_contact')
+                "
+              />
+              <unnnic-tool-tip
+                enabled
+                :text="$t('contact_info.switch_tooltip')"
+                side="bottom"
+                maxWidth="21rem"
+              >
+                <unnnic-icon-svg icon="information-circle-4" scheme="neutral-soft" size="sm" />
+              </unnnic-tool-tip>
+            </div>
+            <unnnic-button-next
+              v-if="!isHistory && !isViewMode"
+              class="transfer__button"
+              :text="$t('contact_info.see_contact_history')"
+              iconLeft="export-1"
+              type="terciary"
               size="small"
-              @click="refreshContactInfos"
-              :disabled="isRefreshContactDisabled"
+              @click="openHistory()"
             />
-          </header>
-
-          <div class="connection-info">
-            <p v-if="room.contact.status === 'online'">
-              {{ $t('status.online') }}
-            </p>
-            <p v-if="lastMessageFromContact?.created_on" style="margin-bottom: 12px">
-              {{
-                $t('last_message_time.date', {
-                  date: moment(lastMessageFromContact?.created_on).fromNow(),
+            <div v-if="isLinkedToOtherAgent">
+              <span>{{
+                $t('contact_info.linked_contact', {
+                  name: this.room.linked_user,
                 })
-              }}
-            </p>
-            <template>
-              <hgroup class="info">
-                <h3 class="title">{{ contactNumber.plataform }}:</h3>
-                <h4 class="description">{{ contactNumber.contactNum }}</h4>
-              </hgroup>
-            </template>
-            <template v-if="!!room.custom_fields">
-              <custom-field
-                v-for="(value, key) in customFields"
-                :key="key"
-                :title="key"
-                :description="value"
-                :is-editable="room.can_edit_custom_fields"
-                :is-current="isCurrentCustomField(key)"
-                :value="currentCustomField?.[key]"
-                @update-current-custom-field="updateCurrentCustomField"
-                @save-value="saveCurrentCustomFieldValue"
-              />
-            </template>
+              }}</span>
+            </div>
+          </section>
+        </aside-slot-template-section>
+
+        <!-- <aside-slot-template-section v-if="isHistory">
```

Comment:
Dead code?

Reply:
No, this code is commented to handle future feature